reana_workflow_engine_serial/tasks.py

```diff
@@ -41,12 +40,30 @@ def escape_shell_arg(shell_arg):
 
     return "%s" % shell_arg.replace('"', '\\"')
 
-
-@app.task(name='tasks.run_serial_workflow',
-          ignore_result=True)
+def load_json(ctx, param, value):
+    """Callback function for click option"""
+    return json.loads(value)
+
+@click.command()
+@click.option('--workflow-uuid',
+              required=True,
+              help='UUID of workflow to be run.')
+@click.option('--workflow-workspace',
+              required=True,
+              help='Name of workspace in which workflow should run.')
+@click.option('--workflow-json',
+              help='JSON representation of workflow object to be run.',
+              callback=load_json)
+@click.option('--workflow-parameters',
+              help='JSON representation of parameters received by'
+                   ' the workflow.',
+              callback=load_json)
+@click.option('--operational-options',
+              help='Options to be passed to the workflow engine'
+                   ' (e.g. caching).',
+              callback=load_json)
 def run_serial_workflow(workflow_uuid, workflow_workspace,
-                        workflow=None, workflow_json=None,
-                        toplevel=os.getcwd(), workflow_parameters=None,
+                        workflow_json=None, workflow_parameters=None,
                         operational_options={}):
```

Comment:
When testing, I saw cases where `operational_options` was being passed `None` value different from `{}`.  It would be therefore safer to do something like:

```diff
diff --git a/reana_workflow_engine_serial/tasks.py b/reana_workflow_engine_serial/tasks.py
index a226972..8403e20 100644
--- a/reana_workflow_engine_serial/tasks.py
+++ b/reana_workflow_engine_serial/tasks.py
@@ -64,8 +64,10 @@ def load_json(ctx, param, value):
               callback=load_json)
 def run_serial_workflow(workflow_uuid, workflow_workspace,
                         workflow_json=None, workflow_parameters=None,
-                        operational_options={}):
+                        operational_options=None):
     """Run a serial workflow."""
+    if not operational_options:
+        operational_options = {}
     workflow_workspace = '{0}/{1}'.format(SHARED_VOLUME_PATH,
                                           workflow_workspace)
     publisher = WorkflowStatusPublisher()
```